api/v1alpha2/jenkins_webhook.go

```diff
@@ -41,21 +48,115 @@ var _ webhook.Validator = &Jenkins{}
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (in *Jenkins) ValidateCreate() error {
-	jenkinslog.Info("validate create", "name", in.Name)
+	if in.Spec.ValidateSecurityWarnings {
+		jenkinslog.Info("validate create", "name", in.Name)
```

Comment:
Are these logs useful in any way?

Reply:
Yeah for debugging purposes I was checking if the function is being called and get a look the the reconciller object.